src/components/CioQuiz/index.tsx

```diff
@@ -29,24 +30,26 @@ export default function CioQuiz(props: IQuizProps) {
     getResetQuizButtonProps,
     getSelectInputProps,
     primaryColorStyles,
-  } = useQuiz(props);
+  } = useQuiz({
+    ...props,
+    enableHydration: enableHydration === undefined ? true : enableHydration,
```

Comment:
If enableHydration isn't passed in the config object default true. if it's passed in the config object use the passed value instead

Reply:
@mocca102 could this logic be moved to useQuiz? What's the benefit of having it here instead of in the hook?